src/main/handlers/projectMetadataHandler.ts

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs/promises';
+import { Project, ProjectMetadata } from '../../sharedTypes';
+
+const retrieveDateModified: (filePath: string) => Promise<Date> = async (
+  filePath
+) => {
+  const fileStats = await fs.stat(filePath);
+
+  return fileStats.mtime;
+};
+
+const retrieveSize: (filePath: string) => Promise<number> = async (
+  filePath
+) => {
+  const fileStats = await fs.stat(filePath);
+
+  return fileStats.size;
+};
+
+const retrieveProjectMetadata: (
+  project: Project
+) => Promise<ProjectMetadata> = async (project) => {
+  const projectLastModified =
+    project.projectFilePath === null
+      ? null
+      : await retrieveDateModified(project.projectFilePath);
+
+  const mediaSize =
+    project.mediaFilePath === null
+      ? null
+      : await retrieveSize(project.mediaFilePath);
+
+  return {
+    dateModified: projectLastModified,
+    mediaSize,
+  };
```

Comment:
Nice work, just a small thing. Think this can be more consistent, like remove `dateModified` or add `mediaSize` to `mediaSize`

Reply:
What do you mean by this? Like consistency between the shorthand and long form? `mediaSize` is just a shorthand for `mediaSize: mediaSize`

Reply:
I think it's fine to leave as is.

Reply:
Yes, and it's fine  to leave as is

Reply:
Yeah in general, if a shorthand is available we should use it, I think eslint is set up to get mad if we don't anyway